src/index.html

```diff
@@ -7,7 +7,58 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+
+    <div class="container">
+
+      <div class="stars stars--0">
```

Comment:
no needs to add an empty line between parent tag and child one
```suggestion
  <body class="body">
    <div class="container">
      <div class="stars stars--0">
```